phpdocker/php-fpm/php-ini-overrides.ini

```diff
@@ -0,0 +1,2 @@
+upload_max_filesize = 100M
```

Comment:
If this configuration necessary? This bundle doesn't upload files, and it won't call PHP-FPM since it will run HP through CLI.

Can it just be removed?

Reply:
I completly removed the ini file. See the comment above:

https://github.com/liip/LiipTestFixturesBundle/pull/260#discussion_r1468435903